MentoringHomeWork/HW6/HW6.Task2/NewArrayPosition.cs

```diff
@@ -0,0 +1,40 @@
+namespace HW6.Task2;
+
+public class NewArrayPosition
```

Comment:
Naming - let's try to rename it on your own.

Reply:
resolved